runtime/mainnet/src/genesis.rs

```diff
@@ -0,0 +1,610 @@
+		derive_multisig::<Runtime>(
+			vec![
+				Keyring::Alice.to_account_id(),
+				Keyring::Bob.to_account_id(),
+				Keyring::Charlie.to_account_id(),
+				Keyring::Dave.to_account_id(),
+			],
+			2,
+		),
+		PARA_ID,
+		// AssetId reserved for DOT from AH.
+		vec![GenesisAsset {
+			id: 0,
+			owner: SudoAddress::get(),
+			is_sufficient: false,
+			min_balance: ExistentialDeposit::get(),
+			name: "DOT".into(),
+			symbol: "DOT".into(),
+			decimals: 10,
+		}],
+		vec![
+			Keyring::Alice.to_account_id(),
+			Keyring::Bob.to_account_id(),
+			Keyring::Charlie.to_account_id(),
+			Keyring::Dave.to_account_id(),
+			Keyring::Eve.to_account_id(),
+		],
+	)
+}
+
+/// Configures a live chain running on multiple nodes on private mainnet, using the `mainnet`
+/// runtime.
+fn live_config() -> Value {
+	let collator_0_account_id: AccountId =
+		AccountId::from_ss58check("15B6eUkXgoLA3dWruCRYWeBGNC8SCwuqiMtMTM1Zh2auSg3w").unwrap();
+	let collator_0_aura_id: AuraId =
+		AuraId::from_ss58check("15B6eUkXgoLA3dWruCRYWeBGNC8SCwuqiMtMTM1Zh2auSg3w").unwrap();
+
+	genesis(
+		// Initial collators.
+		vec![
+			// POP COLLATOR 0
+			(collator_0_account_id, collator_0_aura_id),
+		],
+		vec![],
+		SudoAddress::get(),
+		PARA_ID,
+		vec![],
+		vec![],
+	)
+}
+
+#[allow(clippy::too_many_arguments)]
+fn genesis(
+	invulnerables: Vec<(AccountId, AuraId)>,
+	endowed_accounts: Vec<AccountId>,
+	sudo_key: AccountId,
+	id: ParaId,
+	genesis_assets: Vec<GenesisAsset>,
+	council_members: Vec<AccountId>,
+) -> Value {
+	// Collect genesis assets.
+	// Genesis assets: Vec<(id, owner, is_sufficient, min_balance)>
+	let mut assets: Vec<(u32, AccountId, bool, Balance)> = Vec::new();
+	// Genesis metadata: Vec<(id, name, symbol, decimals)>
+	let mut assets_metadata: Vec<(u32, Vec<u8>, Vec<u8>, u8)> = Vec::new();
+	genesis_assets.iter().for_each(|asset| {
+		assets.push((asset.id, asset.owner.clone(), asset.is_sufficient, asset.min_balance));
+		assets_metadata.push((asset.id, asset.name.clone(), asset.symbol.clone(), asset.decimals));
+	});
+
+	json!({
+		"assets": AssetsConfig {
+			assets,
+			metadata: assets_metadata,
+			..Default::default()
+		},
+		"balances": BalancesConfig { balances: balances(endowed_accounts) },
+		"collatorSelection": {
+			"invulnerables": invulnerables.iter().cloned().map(|(acc, _)| acc).collect::<Vec<_>>(),
+			"candidacyBond": EXISTENTIAL_DEPOSIT * 16,
+		},
+		"council": CouncilConfig {
+			members: council_members,
+			..Default::default()
+		},
+		"parachainInfo": { "parachainId": id },
+		"polkadotXcm": { "safeXcmVersion": Some(SAFE_XCM_VERSION) },
+		"session": {
+			"keys": invulnerables
+				.into_iter()
+				.map(|(acc, aura)| {
+					(
+						acc.clone(),        // account id
+						acc,               	// validator id
+						SessionKeys { aura},// session keys
+					)
+				})
+				.collect::<Vec<_>>(),
+		},
+		"sudo" : { "key" : sudo_key },
+	})
+}
+
+// The initial balances at genesis.
```

Comment:
```suggestion
// The initial balances at genesis for local testing only.
```

might consider adding this as well.

Reply:
[1ea9626](https://github.com/r0gue-io/pop-node/pull/473/commits/1ea962642dad3a32b4908d8f0a501df8fc0e527c)